docker-compose.yml

```diff
@@ -0,0 +1,85 @@
+version: '2.4'
+services:
+  devel:
+    image: local/osbuild-composer-devel
+    build:
+      dockerfile: ./.devcontainer/Dockerfile
```

Comment:
Not sure when this changed but the `context` key seems to be required for some older versions of docker compose.
On Arch with `Docker Compose version 2.2.3` this works.
On F35 with `docker-compose version 1.29.2` the file is considered invalid and should have `context: .` under `build`.

Reply:
Will fix! Thanks :)

Reply:
Fixed.